go.mod

```diff
@@ -1,4 +1,4 @@
-module git.baruwa.io/baruwa-enterprise/rewriteng
+module github.com/kapilth/rewriteng-1
```

Comment:
Will change it to 
github.com/akissa/rewriteng

Reply:
Please fix the module path